gbBuild.py

```diff
@@ -181,6 +182,8 @@
         except:
             row["boundaryCanonical"] = ""
 
+            # why is this one not considered a METADATA ERROR ?
+
```

Comment:
please ignore the autoformatting changes in the first commit. you can view only the comments that I've made by viewing the second commit directly [`463fd79` (#8)](https://github.com/wmgeolab/geoBoundaryBot/pull/8/commits/463fd7958b56b3ec64c088d55c84b8d6cba48edf). I'll remove all of my code comment-questions before this is merged